subgraphs/layer-one/abis/Gravity.json

```diff
@@ -0,0 +1,106 @@
+[
```

Comment:
delete

Reply:
i have to have one abi in the manifest, but i was able to remove everything in between the brackets

Reply:
ah, gotcha. Maybe name it something that makes more sense, like `Placeholder`

Reply:
good idea - done